doc/api/cli.md

```diff
@@ -100,6 +100,128 @@ If this flag is passed, the behavior can still be set to not abort through
 [`process.setUncaughtExceptionCaptureCallback()`][] (and through usage of the
 `node:domain` module that uses it).
 
+### `--allow-child-process`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+When using the [Permission Model][], the process will not be able to spawn any
+child process by default.
+Attemps to do so will throw an `ERR_ACCESS_DENIED` unless the
+user explicitly passes the `--allow-child-process` flag when starting Node.js.
```

Comment:
There should probably be a warning here that allowing child processes (just like native addons) more or less invalidates the entire permission model, and that users must be extremely cautious in how they spawn child processes.

Reply:
I believe the warning would be necessary when the permission model goes out of experimental. There's a warning for the experimental feature, and the flag documentation tries to expose this caution.

In other words, I can include the warning if you feel strongly about it. At this moment, I feel this is optional.

Reply:
That's also what I was talking about, maybe having an extra warning wouldn't hurt?

Reply:
I agree that this needs an additional statement here as suggested by @tniessen. It's a significant foot gun to allow child process when other permissions are denied so it should come with lots of warnings.

Reply:
Alright. Soon as I get back to this code I'll include it.